templates/sub/project_settings/{{ cookiecutter.__folder_name }}/frontend/Dockerfile

```diff
@@ -32,7 +32,7 @@ COPY --from=builder /app/ /app/
 
 RUN <<EOT
     set -e
-    npm i -g corepack@latest && corepack enable pnpm
-    corepack use pnpm@{{ cookiecutter.__version_pnpm }}
-    corepack prepare pnpm@{{ cookiecutter.__version_pnpm }} --activate
+    CI=1 npm i -g corepack@latest && corepack enable pnpm
```

Comment:
@ericof we don't have them in 19 anymore in any of our current projects and works well. And if so, we have to adjust the versions in case it's 18 or 19.